src/olympia/constants/reviewers.py

```diff
@@ -6,14 +6,21 @@
 REVIEWER_REVIEW_LOCK_LIMIT = 3  # How many pages can a reviewer "watch"
 
 # Types of Canned Responses for reviewer tools.
-CANNED_RESPONSE_ADDON = 1
-CANNED_RESPONSE_THEME = 2
-CANNED_RESPONSE_PERSONA = 3
-
-CANNED_RESPONSE_CHOICES = {
-    CANNED_RESPONSE_ADDON: _('Add-on'),
-    CANNED_RESPONSE_THEME: _('Static Theme'),
-    CANNED_RESPONSE_PERSONA: _('Persona'),
+CANNED_RESPONSE_TYPE_ADDON = 1
```

Comment:
nit: it's non-theme rather than addon really because themes are a subset of addons... but sure.

Reply:
I only renamed the constants really